modules/trino/src/main/java/org/testcontainers/containers/TrinoContainer.java

```diff
@@ -0,0 +1,109 @@
+package org.testcontainers.containers;
+
+import org.jetbrains.annotations.NotNull;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.google.common.base.Strings.nullToEmpty;
+import static java.lang.String.format;
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+/**
+ * @author findepi
+ */
+public class TrinoContainer<SELF extends TrinoContainer<SELF>> extends JdbcDatabaseContainer<SELF> {
```

Comment:
let's use the opportunity to fix the SELF reference (not needed here):

```suggestion
public class TrinoContainer extends JdbcDatabaseContainer< TrinoContainer > {
```

Reply:
right, fixed